lib/middleware/karma.js

```diff
@@ -179,6 +179,10 @@ var createKarmaMiddleware = function (
             var filePath = file.path
             var fileExt = path.extname(filePath)
 
+            if (!files.included.hasOwnProperty(i)) {
+              continue
```

Comment:
I think this: https://github.com/karma-runner/karma/pull/2580/files#r104290773 is a better approach to fixing this issue